Makefile

```diff
@@ -22,25 +27,22 @@ TEST_DBUSER=bob
 # This target just saves a bit of typing
 # It takes argument CMD and runs it in the tests service
 run-in-docker:
-	$(docker_env) $(pg2_testsuite_env) docker-compose $(dc_files) run tests $(CMD)
+	$(docker_env) $(testsuite_env) TEST_ASSUMABLE_ARN=$(TEST_ASSUMABLE_ARN) docker-compose $(dc_files) run tests $(CMD)
 
 dc_files=-f docker-compose.yml -f docker-compose.test.yml
 # Enable Buildkit in docker commands
 docker_env=COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1
 
-run-tests-in-docker:  dc-build  # need SSL certs for Postgres services
-	make run-in-docker CMD=""
-
-
 vault_secret = { $\
 "password": "$(TEST_DBPASS)", $\
 "iam_arns": [ $\
-	"${TEST_IAM_ROLE}" $\
+	"${TEST_ASSUMABLE_ARN}", $\
```

Comment:
currently, is TEST_ASSUMABLE_ARN is not provided, there will be an empty string in the iam_arns. Is this a problem?

Reply:
I think that will be fine. It just looks for a match.